src/domain/model.ts

```diff
@@ -107,6 +107,18 @@ const ApprovedAppDocument = new mongoose.Schema(
   { _id: false },
 );
 
+const NotificationSentFlags = new mongoose.Schema(
+  {
+    attestationRequiredNotificationSent: { type: Boolean, required: false },
+    applicationPausedNotificationSent: { type: Boolean, required: false },
+    firstExpiryNotificationSent: { type: Boolean, required: false },
+    secondExpiryNotificationSent: { type: Boolean, required: false },
+    applicationExpiredNotificationSent: { type: Boolean, required: false },
+    applicationClosedNotificationSent: { type: Boolean, required: false },
+  },
+  { _id: false },
+);
+
```

Comment:
Do we have a history somewhere of emails sent?

I'm wondering if we should be capturing the date and maybe an ID of emails that we send of each type so we can trace what we sent... I doubt we keep an email history anywhere so its probably at least somewhat better to capture the date that we sent each email.

Reply:
(ie. make each of these a Date instead of a Boolean)

Reply:
Yeah there isn't a real history anywhere, we do track certain application events like pausing or expiring, but something like the attestation notification wouldn't be tracked...I like the date instead of boolean idea, i will add!